### PR TITLE
Adds craftable firebrands.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Makes fire craftable. Two pieces of wood can be turned into a firebrand, which in turn can be used to light objects.

Also fixes a grammar error in stack crafting.

## Why It's Good For The Game

This is mainly so ooga booga lizards don't have to rely on modern tech to light their pyre.

## Changelog
:cl:
add: You can now craft a firebrand with two pieces of wood.
/:cl:

### CODE REVIEW

**Comment by adamkad1** [comment]
a firebrand is a piece of wood?

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -182,7 +182,7 @@
 		if(!building_checks(R, multiplier))
 			return
 		if (R.time)
-			usr.visible_message("<span class='notice'>[usr] starts building [R.title].</span>", "<span class='notice'>You start building [R.title]...</span>")
+			usr.visible_message("<span class='notice'>[usr] starts building a [R.title].</span>", "<span class='notice'>You start building a [R.title]...</span>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/stacks/stack.dm]: \a

**Comment by HomocidlePancakes** [comment]
You do realise that there is no “proper” way for ooga boogas to get wood?

**Comment by brokenOculus** [comment]
No sprites? How will they be able to pick it up?

**Comment by nemvar** [comment]
The sprite is reusing the match sprite.

@HomocidlePancakes They can grow wood and they have wood in their base.

**Comment by adamkad1** [comment]
but they dont have a hatchet

**Comment by habader** [comment]
You can chop wood with literally anything sharp

**Comment by Akrilla** [comment]
loving these liz buffs

**Comment by nemvar** [comment]
@brokenOculus It reuses the match sprite.

**Pull request merged**